guides/doc-Managing_Hosts/topics/con_overview-of-hosts-in-satellite.adoc

```diff
@@ -4,7 +4,7 @@
 
 A host is any Linux client that {ProjectName} manages.
 Hosts can be physical or virtual.
-Virtual hosts can be deployed on any platform supported by {ProjectName}, such as KVM, VMware vSphere, OpenStack, Amazon EC2, Rackspace Cloud Services or Google Compute Engine.
+Virtual hosts can be deployed on any platform supported by {ProjectName}, such as Amazon EC2, Google Compute Engine, KVM, libvirt, Microsoft Azure, OpenStack, oVirt, Proxmox, Rackspace Cloud Services, RHV, or VMware vSphere.
```

Comment:
```suggestion
Virtual hosts can be deployed on any platform supported by {ProjectName}, such as Amazon EC2, Google Compute Engine, KVM, libvirt, Microsoft Azure, OpenStack, {oVirt}, Proxmox, Rackspace Cloud Services, or VMware vSphere.
```

I'm unsure about Proxmox in Red Hat Satellite @maximiliankolb - let me try and confirm. 
Also, oVirt = RHV https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L53

So if we are to do this, it'll have to be ifdef to remove some of them. Sorry !

Reply:
Great. One more hint if you allow: theforeman.org still shows `RHEV` instead of `RHV`. just fyi.

Rebased with proper attribute for `oVirt`. Also, what would happen if community users use both "oVirt" and "RHV"?

Reply:
> Great. One more hint if you allow: theforeman.org still shows `RHEV` instead of `RHV`. just fyi.

Lol. They've changed their branding and I'm unsure which is the most up to date.
> 
> Rebased with proper attribute for `oVirt`. Also, what would happen if community users use both "oVirt" and "RHV"?

That's a good point. In the docs we only have oVirt for the actual procedures. For the actual computer resource docs, we could add in a note there saying oVirt and RHV are the same things for non-Satellite scenarios. 

For this sentence, we should probably just duplicate with an ifdef here for Satellite with the original sentence, and then ifndef Satellite for the rest :-)

Reply:
I think a note somewhere would not hurt.

Regarding this sentence: I am currently working on another PR to add a sentence for `orcharhino` only. I can rebase afterwards of course.